### PR TITLE
Add light entity

### PR DESCRIPTION
**Pull request by DerenokDmitry**
Added light entity
Added properties for mysql

### CODE REVIEW

File: src/main/kotlin/smart/home/project/controller/LightController.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    fun findByName(@PathVariable name: String) = lightService.findLightByName(name)
+
+    @GetMapping("/{status}")
+    fun findByLight(@PathVariable status: Boolean) = lightService.findLightByStatus(status)
+
+    @PostMapping
+    fun createLight(@RequestBody light: Light) = lightService.createLight(light)
+
+    @PutMapping("/{id}")
+    fun changeLightStatus(@PathVariable id: Long) = lightService.changeLightStatus(id)
```
Comment (Leveoran) [inline src/main/kotlin/smart/home/project/controller/LightController.kt]: Rename method "change..." to "update" because this is put mapping

Comment (Leveoran) [reply src/main/kotlin/smart/home/project/controller/LightController.kt]: And same in Service

File: src/main/kotlin/smart/home/project/model/Light.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType.AUTO
+import javax.persistence.Id
+
+@Entity
+class Light(
+    @Id
+    @GeneratedValue(strategy = AUTO)
```
Comment (Leveoran) [inline src/main/kotlin/smart/home/project/model/Light.kt]: This is default value, can be deleted

File: src/main/kotlin/smart/home/project/service/LightService.kt [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    fun findLightByStatus(status: Boolean) = lightRepository.findByLightStatus(status)
+
+    @Transactional
+    fun createLight(light: Light) = lightRepository.save(light)
+
+    @Transactional
+    fun changeLightStatus(id: Long) {
+        val light = lightRepository.findById(id)
+        light.get().lightStatus = light.get().lightStatus
```
Comment (Leveoran) [inline src/main/kotlin/smart/home/project/service/LightService.kt]: Why do u don't change name when update?

File: src/main/kotlin/smart/home/project/controller/LightController.kt (hunk 1 above)
Comment (Leveoran) [inline src/main/kotlin/smart/home/project/controller/LightController.kt]: Is Put mapping needed here? Podumai epta

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -1 +1,4 @@
-
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:mysql://localhost:3306/smart_home
+spring.datasource.username=root
```
Comment (Leveoran) [inline src/main/resources/application.properties]: Move username and password to environment variables

File: src/main/kotlin/smart/home/project/service/LightService.kt (hunk 3 above)
Comment (DerenokDmitry) [reply src/main/kotlin/smart/home/project/service/LightService.kt]: Update light name will in future feature

File: src/main/resources/application.properties (hunk 4 above)
Comment (DerenokDmitry) [reply src/main/resources/application.properties]: In the future

**Pull request merged**